### DISCUSSION TITLE
blog/kehidupan-setelah-smk

### DISCUSSION THREAD

**Discussion by giscus** [hendraaagil/website#22] [Comments]
# blog/kehidupan-setelah-smk

Apa yang akan kita hadapi setelah masa SMK? Yaitu sebuah kehidupan yang sebenarnya :)

https://hendraaagil.dev/blog/kehidupan-setelah-smk

**Comment by Rychhhh** [comment]
gud bro

**Comment by RizkiBinyola** [comment]
Hosting dimana bang?

**Reply by hendraaagil** [reply]
Vercel bang

**Reply by RizkiBinyola** [reply]
vercel itu untuk web static bang?

**Reply by hendraaagil** [reply]
iya bang

**Reply by RizkiBinyola** [reply]
trus ini abang bikin blog databasenya gimana? sorry nanya nanya bang soalnya saya ada niatan ingin buat blog seperti ini dan saya juga lagi pengin belajar react dan node js 🙏

**Reply by hendraaagil** [reply]
blognya cuma pake markdown, ada di https://github.com/hendraaagil/website/tree/main/src/_blogs

**Reply by RizkiBinyola** [reply]
ini mirip kaya jekyll atau hugo kah bang?

**Reply by hendraaagil** [reply]
wah kurang tau ya, belum pernah coba keduanya

**Reply by RizkiBinyola** [reply]
ada tutorialnya gak bang untuk buat website seperti ini? untuk belajar nanti

**Reply by hendraaagil** [reply]
belum ada sih, ini aku dari baca dokumentasi library-nya aja

**Reply by RizkiBinyola** [reply]
oy bang ini kan blognya make markdown trus cara masukin giscus nya gimana?

**Reply by hendraaagil** [reply]
tinggal panggil componentnya aja di tiap page
https://github.com/hendraaagil/website/blob/main/src/pages/blog/%5Bslug%5D.tsx#L95
